client/main.lua

```diff
@@ -377,6 +383,64 @@ RegisterNUICallback("incidentSearchPerson", function(data, cb)
     cb(true)
 end)
 
+-- Handle sending the player to jail
+-- Uses QB-Core/qb-policejob function to send the player to jail
+-- If you use a different jail system, you will need to change this
+RegisterNUICallback("sendToJail", function(data, cb)
+    local citizenId, sentence = data.citizenId, data.sentence
+
+    -- Gets the player id from the citizenId
+    local p = promise.new()
+    QBCore.Functions.TriggerCallback('mdt:server:GetPlayerSourceId', function(result)
+        p:resolve(result)
+    end, citizenId)
+
+    local targetSourceId = Citizen.Await(p)

```

Comment:
The UI only has the citizenId for who we are performing the action on. We need a way to convert that citizenId to a serverId so that we can perform the actions. This uses a simple callback that will get the sourceId for the given citizenId. 

I realize this isn't the prettiest and there may be a simpler way to achieve this. Open to suggestions